src/theme.ts

```diff
@@ -1,14 +1,41 @@
 /* eslint-disable @mll-lab/no-color-literals */
 
+import {
+  red,
+  volcano,
+  orange,
+  gold,
+  yellow,
+  lime,
+  green,
+  cyan,
+  blue,
+  geekblue,
+  purple,
+  magenta,
+  grey,
+} from '@ant-design/colors';
 import { useContext } from 'react';
 import { ThemeContext } from 'styled-components';
 
-/**
- * We omit special purpose colors:
- * - 'white': Brightest possible
- * - 'black': Darkest possible
- * - 'transparent': No color
- */
+export const MLL_PALETTE = {
+  red: red.primary,
+  volcano: volcano.primary,
+  orange: orange.primary,
+  gold: gold.primary,
+  yellow: yellow.primary,
+  lime: lime.primary,
+  green: green.primary,
+  cyan: cyan.primary,
+  blue: blue.primary,
+  geekblue: geekblue.primary,
+  purple: purple.primary,
+  magenta: magenta.primary,
+  grey: grey.primary,
+  white: '#ffffff',
+  black: '#000000',
+};
+
 export const MLL_THEME = {
```

Comment:
```suggestion
export const THEME = {
```

Reply:
yes

Reply:
I've added a deprecated copy of THEME so that we have no stress to replace the old usages..